STYLE.md

```diff
@@ -0,0 +1,270 @@
+#Telia Company Terraform Style Guide
+
+## Introduction
+
+This guide gives developers a guidance in the coding conventions used by Telia Company  for Terraform's HashiCorp Configuration Language (HCL). Terraform allows infrastructure to be described as code, which in turns makes changes and implementations of readable in a structured way. As such, we should adhere to a style guide to ensure readable and high quality code. As Terraform utilises [HCL](https://github.com/hashicorp/hcl), you may wish to take a detailed look at its [syntax guide](https://github.com/hashicorp/hcl/blob/master/README.md#syntax) for further guidance.
+
+Inspired by [The Ruby Style Guide](https://github.com/bbatsov/ruby-style-guide) and [The Puppet Style Guide](https://docs.puppetlabs.com/guides/style_g uide.html).
+
+## Syntax
+
+- Strings are in double-quotes.
+
+### Spacing
+
+Use 2 spaces when defining resources except when defining inline policies or other inline resources.
+
+```
+resource "aws_iam_role" "iam_role" {
+  name = "${var.resource_name}-role"
+  assume_role_policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": "sts:AssumeRole",
+      "Principal": {
+        "Service": "ec2.amazonaws.com"
+      },
+      "Effect": "Allow",
+      "Sid": ""
+    }
+  ]
+}
+EOF
+}
+```
+
+There should be two blank lines between resource definitions.
```

Comment:
I think `terraform fmt` will remove the extra whitespace in this case. So it should just be one blank line between resource definitions.